driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudHelper.java

```diff
@@ -90,6 +91,8 @@
 
 final class UnifiedCrudHelper {
     private final Entities entities;
+    private final AtomicInteger uniqueId = new AtomicInteger();
```

Comment:
[optional]
It's an ID generator, not an ID. Let's rename it to facilitate readability.

Reply:
Done